### DISCUSSION TITLE
Welcome to gsoc-contrib Discussions!

### DISCUSSION THREAD

**Discussion by mdubinko** [mdubinko/gsoc-contrib#8] [Announcements]
## 👋 Welcome!

I'm new at this, but let's have a place to collaborate and strategize.

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ricard63** [comment]
Hello there,
I have recently had the GSOC, and am experimenting a bit with it.
At the moment I have seen that the firmware installation SDs start perfectly in my CubieAIO-A20 and stop when they do not find something. The system accepts orders, but will not follow..

I have also extracted the contents of the SD, which from what I have seen contains several partitions.

I would like to collaborate with this project

**Comment by NeonFux** [comment]
https://github.com/d3cker/gsoc-puhumod
Debian on GSOC

**Comment by mdubinko** [comment]
*applause* Nice work.

**Comment by NeonFux** [comment]
https://github.com/orgs/cubieboard/repositories?type=all

**Comment by wwashing** [comment]
I'm surprised to find out that there is no ongoing support for the GSOC.  I'm considering "jailbreak" for my GSOC ... has anyone tried
this?
-Next Step-
